EE Acc Tests Failing on TestAccGitlabProjectPushRules_import

The Gitlab EE acceptance tests are failing in `master`, after the new EE license was added.

https://github.com/gitlabhq/terraform-provider-gitlab/runs/1089051756

```
2020-09-09T02:44:31.0605132Z === RUN   TestAccGitlabProjectPushRules_import
<snip>
2020-09-09T02:44:31.4408000Z 2020/09/09 02:44:31 [ERROR] <root>: eval: *terraform.EvalImportState, err: GET http://127.0.0.1:8080/api/v4/projects/foo-8362209985828888294/push_rule: 404 {message: 404 Project Not Found}
2020-09-09T02:44:31.4423981Z 2020/09/09 02:44:31 [ERROR] <root>: eval: *terraform.EvalSequence, err: GET http://127.0.0.1:8080/api/v4/projects/foo-8362209985828888294/push_rule: 404 {message: 404 Project Not Found}
2020-09-09T02:44:31.4424713Z 2020/09/09 02:44:31 [ERROR] Test: ImportState failure: GET http://127.0.0.1:8080/api/v4/projects/foo-8362209985828888294/push_rule: 404 {message: 404 Project Not Found}
2020-09-09T02:44:31.4429518Z ##[error]    testing.go:684: Step 1 error: GET http://127.0.0.1:8080/api/v4/projects/foo-8362209985828888294/push_rule: 404 {message: 404 Project Not Found}
<snip>
2020-09-09T02:44:36.9397185Z --- FAIL: TestAccGitlabProjectPushRules_import (6.53s)
```

I'm trying a fix in https://github.com/gitlabhq/terraform-provider-gitlab/tree/acc-ee-fix now.

--- a/gitlab/resource_gitlab_project_push_rules.go
+++ b/gitlab/resource_gitlab_project_push_rules.go
@@ -3,6 +3,7 @@ package gitlab
 import (
 	"fmt"
 	"log"
+	"strconv"
 
 	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
 	gitlab "github.com/xanzy/go-gitlab"
@@ -152,7 +153,7 @@ func resourceGitlabProjectPushRulesImporter(d *schema.ResourceData, meta interfa
 	d.SetId(fmt.Sprintf("%d", pushRules.ID))
 
 	// Since project is used as a primary key in the Read function, we set that too.
-	d.Set("project", project)
+	d.Set("project", strconv.Itoa(pushRules.ProjectID))
 
 	return []*schema.ResourceData{d}, nil
 }

--- a/gitlab/resource_gitlab_project_push_rules_test.go
+++ b/gitlab/resource_gitlab_project_push_rules_test.go
@@ -93,7 +93,7 @@ func TestAccGitlabProjectPushRules_import(t *testing.T) {
 			{
 				SkipFunc:          isRunningInCE,
 				ResourceName:      "gitlab_project_push_rules.foo",
-				ImportStateId:     fmt.Sprintf("foo-%d", rInt),
+				ImportStateId:     fmt.Sprintf("root/foo-%d", rInt),
 				ImportState:       true,
 				ImportStateVerify: true,
 			},

